tests/testthat/test_impute.R

```diff
@@ -0,0 +1,67 @@
+test_that("Testing .impute function", {
```

Comment:
can you be a bit more specific? it is ok to have more than on test_that() per file